libraries/abstractions/secure_sockets/test/iot_test_tcp.c

```diff
@@ -2961,6 +2962,61 @@ TEST( Full_TCP, AFQP_SOCKETS_htons_HappyCase )
 }
 /*-----------------------------------------------------------*/
 
+TEST( Full_TCP, test_dns_cache_multiple_addresses )
```

Comment:
Is this test verifying DNS caching? The implementation of `SOCKETS_GetHostByName` may not use DNS caching and instead rely on DNS round robin- the test would still pass?

Reply:
Yes, that's a good point.  This wouldn't be the case in the FreeRTOS+TCP implementation, but it could be in others.  This test will be disabled in all platforms which use non-FreeRTOS+TCP networking until we can verify that it works on them.  In fact, having the word `cache` in this test name may be misleading, since it's not verifying that a DNS cache is enabled (I'm not sure if this would even be possible in this test), only that different IP addresses are returned for different invocations of `SOCKETS_GetHostByName()`.  I will rename the test to something more accurate, perhaps `test_dns_multiple_addresses`.